### DISCUSSION TITLE
Галочка Debug в настройках?

### DISCUSSION THREAD

**Discussion by fishenebelny** [TionAPI/HA-tion#78] [Ideas]
Может быть добавить в настройки бризера галочку Debug. Ну чтобы совсем красиво было. Чтобы не лезть в configuration.yaml?

**Comment by IATkachenko** [comment]
Уровень логов задается именно в HA, интеграция на это влиять не может, как я понимаю. А дублировать механизм логирования -- это не то что нужно/хочется делать.

То что в некоторых интеграциях появилась кнопка diagnostic info -- это интеграция сама о себе собирает существенную информацию, без вмешательства в ядро. Только то, до чего сама может дотянуться.

Я еще гляну какие возможности в плане изменения уровня логов доступны на уровне интеграции, но не думаю что из этого что-то выйдет полезное.
